testng-core/src/test/java/test/listeners/issue2685/SampleInterruptedTest.java

```diff
@@ -0,0 +1,16 @@
+package test.listeners.issue2685;
+
+import org.testng.annotations.Test;
+
+public class SampleInterruptedTest {
```

Comment:
For the samples that are being used in the tests, we are trying to follow a nomenclature of having the test class end with `Sample`. So please help rename this class to `InterruptedTestSample`

Reply:
Done